lib/actions/preload.coffee

```diff
@@ -159,8 +160,8 @@ module.exports =
 			signature: 'commit'
 			parameter: 'hash'
 			description: '''
-				the commit hash for a specific application release to preload, use "latest" to specify the latest release
-				(ignored if no appId is given)
+				the commit hash for a specific application release to preload, use "current" (or "latest") to specify
+				the current release (ignored if no appId is given)
```

Comment:
I'm debating with myself whether mentioning "latest" does more harm than good... I expect that the vast majority of users will wonder what "current" means exactly, and mentioning "or latest" could hint that "current" means "most recent" (but it doesn't). I've thought of the following description:
```
the commit hash for a specific application release to preload. Use "current"
to refer to the current value of the API Application object's commit field.
(ignored if no appId is given)
```
Maybe we could have more feedback from people like... @imrehg and @dfunckt

Reply:
I agree, the term "current" is more accurate and it just happens that usually also means "latest". This may be a good clarification to make. For example:

> [...] the commit hash for a specific application release to preload, use "current" to specify the current release (ignored if no appId is given). The current release is usually also the latest, but can be manually pinned using the CLI to set the `commit` field on the application.